packages/client/hmi-client/src/workflow/ops/model-from-code/tera-model-from-code-node.vue

```diff
@@ -1,5 +1,14 @@
 <template>
 	<main>model from code node</main>
+	<tera-operator-image-carousel :images="images" />
```

Comment:
Could we use `<slot>` instead?
I would prefer to look like the following examples. Like `tera-operator-image-carousel` is to make a list of images looks better and add functionality to a list of images.
```vue
<tera-operator-image-carousel>
  <img src="@/assets/svg/terarium-icon.svg" alt="Terarium Icon" />
  <img src="@/assets/svg/uncharted-logo-official.svg" alt="Uncharted Logo" />
  <img src="@/assets/svg/terarium-logo.svg" alt="Terarium Logo" />
</tera-operator-image-carousel>
```
Or if the dev prefers JS
```vue
<tera-operator-image-carousel>
  <img v-for="image in images" :src="image.src" :alt="image.alt" />
</tera-operator-image-carousel>
```
If this is too complex or difficult, then don't worry.

Reply:
Yeah this is getting into fighting with primevue territory since `Galleria` expects a value and it uses the `#item` template.

Reply:
Too bad!